[2025-02-13T23:21:07.721+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_dataset.convert_to_parquet manual__2025-02-13T23:20:50.726546+00:00 [queued]>
[2025-02-13T23:21:07.731+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_dataset.convert_to_parquet manual__2025-02-13T23:20:50.726546+00:00 [queued]>
[2025-02-13T23:21:07.732+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-13T23:21:07.745+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonVirtualenvOperator): convert_to_parquet> on 2025-02-13 23:20:50.726546+00:00
[2025-02-13T23:21:07.751+0000] {standard_task_runner.py:57} INFO - Started process 1131 to run task
[2025-02-13T23:21:07.754+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'download_dataset', 'convert_to_parquet', 'manual__2025-02-13T23:20:50.726546+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/download_dataset_dag.py', '--cfg-path', '/tmp/tmphoa7c99p']
[2025-02-13T23:21:07.755+0000] {standard_task_runner.py:85} INFO - Job 98: Subtask convert_to_parquet
[2025-02-13T23:21:07.805+0000] {task_command.py:416} INFO - Running <TaskInstance: download_dataset.convert_to_parquet manual__2025-02-13T23:20:50.726546+00:00 [running]> on host 4f1c81a800b8
[2025-02-13T23:21:07.885+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='download_dataset' AIRFLOW_CTX_TASK_ID='convert_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T23:20:50.726546+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-13T23:20:50.726546+00:00'
[2025-02-13T23:21:07.886+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv7342py_x
[2025-02-13T23:21:07.897+0000] {process_utils.py:186} INFO - Output:
[2025-02-13T23:21:09.000+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 255ms
[2025-02-13T23:21:09.001+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venv7342py_x, clear=False, no_vcs_ignore=False, global=False)
[2025-02-13T23:21:09.001+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-02-13T23:21:09.001+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.2.1, setuptools==68.2.0, wheel==0.41.2
[2025-02-13T23:21:09.001+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-02-13T23:21:09.042+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv7342py_x/bin/pip install -r /tmp/venv7342py_x/requirements.txt
[2025-02-13T23:21:09.050+0000] {process_utils.py:186} INFO - Output:
[2025-02-13T23:21:10.955+0000] {process_utils.py:190} INFO - Collecting pandas (from -r /tmp/venv7342py_x/requirements.txt (line 1))
[2025-02-13T23:21:10.955+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pandas from https://files.pythonhosted.org/packages/f8/7f/5b047effafbdd34e52c9e2d7e44f729a0655efafb22198c45cf692cdc157/pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2025-02-13T23:21:10.957+0000] {process_utils.py:190} INFO -   Using cached pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (18 kB)
[2025-02-13T23:21:11.169+0000] {process_utils.py:190} INFO - Collecting pyarrow (from -r /tmp/venv7342py_x/requirements.txt (line 2))
[2025-02-13T23:21:11.170+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pyarrow from https://files.pythonhosted.org/packages/e6/c1/4c6bcdf7a820034aa91a8b4d25fef38809be79b42ca7aaa16d4680b0bbac/pyarrow-17.0.0-cp38-cp38-manylinux_2_28_x86_64.whl.metadata
[2025-02-13T23:21:11.172+0000] {process_utils.py:190} INFO -   Using cached pyarrow-17.0.0-cp38-cp38-manylinux_2_28_x86_64.whl.metadata (3.3 kB)
[2025-02-13T23:21:11.296+0000] {process_utils.py:190} INFO - Collecting python-dateutil>=2.8.2 (from pandas->-r /tmp/venv7342py_x/requirements.txt (line 1))
[2025-02-13T23:21:11.297+0000] {process_utils.py:190} INFO -   Obtaining dependency information for python-dateutil>=2.8.2 from https://files.pythonhosted.org/packages/ec/57/56b9bcc3c9c6a792fcbaf139543cee77261f3651ca9da0c93f5c1221264b/python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata
[2025-02-13T23:21:11.298+0000] {process_utils.py:190} INFO -   Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
[2025-02-13T23:21:11.382+0000] {process_utils.py:190} INFO - Collecting pytz>=2020.1 (from pandas->-r /tmp/venv7342py_x/requirements.txt (line 1))
[2025-02-13T23:21:11.383+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pytz>=2020.1 from https://files.pythonhosted.org/packages/eb/38/ac33370d784287baa1c3d538978b5e2ea064d4c1b93ffbd12826c190dd10/pytz-2025.1-py2.py3-none-any.whl.metadata
[2025-02-13T23:21:11.385+0000] {process_utils.py:190} INFO -   Using cached pytz-2025.1-py2.py3-none-any.whl.metadata (22 kB)
[2025-02-13T23:21:11.422+0000] {process_utils.py:190} INFO - Collecting tzdata>=2022.1 (from pandas->-r /tmp/venv7342py_x/requirements.txt (line 1))
[2025-02-13T23:21:11.423+0000] {process_utils.py:190} INFO -   Obtaining dependency information for tzdata>=2022.1 from https://files.pythonhosted.org/packages/0f/dd/84f10e23edd882c6f968c21c2434fe67bd4a528967067515feca9e611e5e/tzdata-2025.1-py2.py3-none-any.whl.metadata
[2025-02-13T23:21:11.424+0000] {process_utils.py:190} INFO -   Using cached tzdata-2025.1-py2.py3-none-any.whl.metadata (1.4 kB)
[2025-02-13T23:21:11.929+0000] {process_utils.py:190} INFO - Collecting numpy>=1.20.3 (from pandas->-r /tmp/venv7342py_x/requirements.txt (line 1))
[2025-02-13T23:21:11.930+0000] {process_utils.py:190} INFO -   Obtaining dependency information for numpy>=1.20.3 from https://files.pythonhosted.org/packages/98/5d/5738903efe0ecb73e51eb44feafba32bdba2081263d40c5043568ff60faf/numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2025-02-13T23:21:11.933+0000] {process_utils.py:190} INFO -   Using cached numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.6 kB)
[2025-02-13T23:21:11.983+0000] {process_utils.py:190} INFO - Collecting six>=1.5 (from python-dateutil>=2.8.2->pandas->-r /tmp/venv7342py_x/requirements.txt (line 1))
[2025-02-13T23:21:11.984+0000] {process_utils.py:190} INFO -   Obtaining dependency information for six>=1.5 from https://files.pythonhosted.org/packages/b7/ce/149a00dd41f10bc29e5921b496af8b574d8413afcd5e30dfa0ed46c2cc5e/six-1.17.0-py2.py3-none-any.whl.metadata
[2025-02-13T23:21:11.986+0000] {process_utils.py:190} INFO -   Using cached six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
[2025-02-13T23:21:12.037+0000] {process_utils.py:190} INFO - Using cached pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.4 MB)
[2025-02-13T23:21:12.188+0000] {process_utils.py:190} INFO - Using cached pyarrow-17.0.0-cp38-cp38-manylinux_2_28_x86_64.whl (40.0 MB)
[2025-02-13T23:21:12.304+0000] {process_utils.py:190} INFO - Using cached numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
[2025-02-13T23:21:12.332+0000] {process_utils.py:190} INFO - Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
[2025-02-13T23:21:12.335+0000] {process_utils.py:190} INFO - Using cached pytz-2025.1-py2.py3-none-any.whl (507 kB)
[2025-02-13T23:21:12.337+0000] {process_utils.py:190} INFO - Using cached tzdata-2025.1-py2.py3-none-any.whl (346 kB)
[2025-02-13T23:21:12.341+0000] {process_utils.py:190} INFO - Using cached six-1.17.0-py2.py3-none-any.whl (11 kB)
[2025-02-13T23:21:12.828+0000] {process_utils.py:190} INFO - Installing collected packages: pytz, tzdata, six, numpy, python-dateutil, pyarrow, pandas
[2025-02-13T23:21:20.971+0000] {process_utils.py:190} INFO - Successfully installed numpy-1.24.4 pandas-2.0.3 pyarrow-17.0.0 python-dateutil-2.9.0.post0 pytz-2025.1 six-1.17.0 tzdata-2025.1
[2025-02-13T23:21:21.066+0000] {process_utils.py:190} INFO - 
[2025-02-13T23:21:21.066+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.2.1 -> 25.0.1
[2025-02-13T23:21:21.066+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-02-13T23:21:21.243+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv7342py_x/bin/python /tmp/venv7342py_x/script.py /tmp/venv7342py_x/script.in /tmp/venv7342py_x/script.out /tmp/venv7342py_x/string_args.txt /tmp/venv7342py_x/termination.log
[2025-02-13T23:21:21.253+0000] {process_utils.py:186} INFO - Output:
[2025-02-13T23:21:22.397+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-13T23:21:22.410+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=download_dataset, task_id=convert_to_parquet, execution_date=20250213T232050, start_date=20250213T232107, end_date=20250213T232122
[2025-02-13T23:21:22.443+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-13T23:21:22.465+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
